RNTester/js/examples/Pressable/PressableExample.js

```diff
@@ -54,6 +54,34 @@ function ContentPress() {
   );
 }
 
+function TVFocusContentPress() {
+  const [timesPressed, setTimesPressed] = useState(0);
+
+  let textLog = '';
+  if (timesPressed > 1) {
+    textLog = timesPressed + 'x onPress';
+  } else if (timesPressed > 0) {
+    textLog = 'onPress';
+  }
+
+  return (
+    <>
+      <View style={styles.row}>
+        <Pressable
```

Comment:
might be useful to actually show the usage of onFocus/onBlur in the example here - eg.:
onBlur={() => console.log('Pressable onBlur')} or sth making  a change to the UI.